packages/toggle-crud-psr11-factories/test/PostFeatureFactoryTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Pheature\Test\Crud\Psr11\Toggle;
+
+use Pheature\Core\Toggle\Write\FeatureRepository;
+use Pheature\Crud\Psr11\Toggle\PostFeatureFactory;
+use Pheature\Crud\Psr7\Toggle\PostFeature;
+use Pheature\Crud\Toggle\Handler\CreateFeature;
+use PHPUnit\Framework\TestCase;
+use Prophecy\PhpUnit\ProphecyTrait;
+use Psr\Container\ContainerInterface;
+use Psr\Http\Message\ResponseFactoryInterface;
+
+final class PostFeatureFactoryTest extends TestCase
+{
+    use ProphecyTrait;
+
+    public function testItShouldCreateAPostFeatureFromInvokable(): void
```

Comment:
Take a look at this PR https://github.com/pheature-flags/pheature-flags/pull/147/files, is the same case but with PHPUnit, I don't see a real gain here if we can't mock the final classes.

To avoid adding Mock's "method" method, we need a trait, to reveal mocks, and to learn a different nomenclature for expectations.

I think that we can continue the discussion in #154

Reply:
You're right, I think that too. I'm going to remove the dependency and right now, we can use the PhpUnit mock and try to find a better strategy! Thanks :)